cosmwasm/packages/wasmi-runtime/src/registration/offchain.rs

```diff
@@ -1,21 +1,30 @@
+#[cfg(feature = "SGX_MODE_HW")]
+use enclave_ffi_types::NodeAuthResult;
 ///
 /// These functions run off chain, and so are not limited by deterministic limitations. Feel free
 /// to go crazy with random generation entropy, time requirements, or whatever else
```

Comment:
please put this import below the docstring, with an empty line between them, and then replace the `///` with `//!` so it's correctly applied to the module

Reply:
Ah, cool. The comment was moved by the CLion auto optimize imports thing. With //! it properly leaves it there